trulens_eval/trulens_eval/instruments.py

```diff
@@ -1020,3 +1020,43 @@ def instrument_object(self, obj, query: Query, done: Set[int] = None):
             logger.debug(
                 f"{query}: Do not know how to instrument object of type {cls}."
             )
+
+
+class instrument:
```

Comment:
nit: naming capitalized?  Maybe name InstrumentedDecorator

Reply:
this one is used like a method as in `@instrument` . I did however separate out the utilities not for decoration from this "class" and named it with a capital `AddInstruments`.